src/common/transformations/include/transformations/common_optimizations/markup_rope_inputs.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+/**
+ * @ingroup ov_transformation_common_api
+ * @brief This transformation markups the 2nd/3rd inputs of Rope with FP32 to mantian accuracy.
+ * +-------+    +-------+    +-------+
+ * |intput1|    |input2 |    |input3 |
+ * |(orig) |    |(fp32) |    |(fp32) |
+ * +---|---+    +---|---+    +---|---+
+ *     |            |            |
+ *     |            |            |
+ *  +--+------------|------------+--+
+ *  |                               |
+ *  |             ROPE              |
+ *  +-------------------------------+
+ */
+
+class TRANSFORMATIONS_API MarkUpRopeInputs : public ov::pass::MatcherPass {
```

Comment:
I guess it should be named "MarkRopeInputsToKeepInMixedPrecision" to align with "MarkSugraphsToKeepInMixedPrecision".
Just "MarkUp" sounds to generic